.gitignore

```diff
@@ -3,3 +3,4 @@
 *.pyc
 __pycache__/
 data/
+!data/.gitkeep
```

Comment:
what does this do?

Reply:
oops. That was supposed to include the directory with no files in it so the lack of directory wouldn't break the csv writer. I think I forgot to add that file, though.